projects/abseil-cpp/project.yaml

```diff
@@ -0,0 +1,3 @@
+homepage: "abseil.io"
+language: c++
+primary_contact: "todo@gmail.com"
```

Comment:
let's add a proper contact here. @derekmauro is there someone from abseil who would be best here?

Reply:
abseil-io@googlegroups.com